### ISSUE TITLE
Too much fog

### ISSUE THREAD

**Issue by Danogot516** [yvt/openspades#427]
I can't seem to deal with the amount of fog there is.And Ace of Spades Classic there wasn't so much fog and I've tried disabling "Depth Fog" which is r_fogShadow in the advanced settings but that doesn't help.

**Comment by IBPX** [comment]
I agree, though I believe servers choose how much fog there is. If you play the same server in OpenSpades vs. Classic, is the fog different? (possibly post screenshots)

**Comment by Kurtoid** [comment]
The fog is close to keep the maps small enough to keep people nearby, but
still allow flanking and other manuvers.

**Comment by pielover88888** [comment]
Kurtoid: I think the issue is that we're playing on the same servers with different fog levels - with classic having less fog. It's also worth noting classic doesn't have any vertical fog whatsoever. Servers don't choose the fog distance, it's entirely done clientside (if we're talking about aos 0.75 and openspades on pysnip servers).

**Comment by Danogot516** [comment]
Yeah, we tested it with a friend of mine who was using the ace of spades 0.75 and me, using openspades client.He could see further than I could.That's really irritating when you don't see someone infront of you because of the fog but he sees you clearly.

**Comment by Danogot516** [comment]
So what can we do to fix this????

**Comment by Phyllostachys** [comment]
Maybe this is related to #409?

**Comment by Danogot516** [comment]
It is but there isn't any update on that thread and this fog is pissing me off.

**Comment by Danogot516** [comment]
So now what?Should we wait for yvt to see the thread?

**Comment by Fizzelo** [comment]
Fog is still shit. This client is useless until fixed.
